libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -103,9 +114,42 @@ export default function TeamCharacterSelector({
         }
       >
         <CardContent sx={{ display: 'flex', justifyContent: 'center', pb: 0 }}>
-          <Typography variant="h5" display="flex">
-            {team.name}
+          <Typography
```

Comment:
Can you wrap this in a `<CardActionArea>` so the element is highlighted when hovered to better indicate interactivity?

Reply:
Yea, I can do that. Also, do you have any specifics when it comes to UI. Was thinking about adding a max-width to the `<CardActionArea>` so that it doesn't take up the whole space at the top.

Reply:
probably not necessary, just make the entire title width interactable is good enough.